src/SOS/SOS.UnitTests/SOS.cs

```diff
@@ -54,18 +54,29 @@ private async Task RunTest(TestConfiguration config, string testName, string deb
             await runner.RunScript(scriptName);
         }
 
-        // Against a crash dump.
+        // Generate a crash dump.
         if (IsCreateDumpConfig(config))
         {
             await SOSRunner.CreateDump(config, Output, testName, debuggeeName, debuggeeArguments, useCreateDump);
         }
 
+        // Test against a crash dump.
         if (IsOpenDumpConfig(config))
         {
-            using (SOSRunner runner = await SOSRunner.StartDebugger(config, Output, testName, debuggeeName, debuggeeArguments, loadDump: true))
+            // With cdb (Windows) or lldb (Linux or OSX)
+            using (SOSRunner runner = await SOSRunner.StartDebugger(config, Output, testName, debuggeeName, debuggeeArguments, SOSRunner.Options.LoadDump))
             {
                 await runner.RunScript(scriptName);
             }
+
+            // With the dotnet-dump analyze tool
+            if (OS.Kind == OSKind.Linux)
```

Comment:
It currently only supported on Linux.  I haven't got the Windows support done yet.  And MacOS will have to wait for a macho core file reader.

Reply:
Thanks Mike. If there isn't something that explicitly blocks people from trying to use it on OSX + Windows I think we should add that, but no need for it to be in this PR.

Reply:
Maybe I'm missunderstanding your point. We can only test dotnet-dump on Linux.  It  doesn't work on WIndows or OSX.  If that if wasn't there, the tests would crash/fail.

Reply:
Sorry, I wasn't referring to the testing in my reply, I was referring to whether something is in the product for dotnet analyze that will prevent people from running it on Windows and OSX. My goal was to see something like this on Windows:

    > dotnet dump analyze foo.dmp
    Preview build: This command is not yet implemented on Windows

The idea being that if we are making Preview builds and telling people to try it, then they are likely to try things we haven't finished yet. I wasn't sure if the experience already looked like that, but if not I was encouraging that we should get that explicit blocking in there.

Reply:
I understand now.  For OSX, it is something like that. I need a better error message.

For Windows, it actually loads the minidump and the "lm" and "threads" native state commands work, but as soon as a SOS command is execute, there is an error message (again not a good one).  I didn't completely disable it on Windows because I was using it to debug the command processor, etc.  I probably should add a error message on start and not allow it to get to the command prompt.